tests/test_downloadermiddleware_httpcache.py

```diff
@@ -14,7 +14,7 @@
 
 
 class _BaseTest(unittest.TestCase):
-    storage_class = "scrapy.extensions.httpcache.DbmCacheStorage"
+    storage_class = "scrapy.extensions.httpcache.FilesystemCacheStorage"
```

Comment:
I wonder if it'd be better to move these constants to DefaultStorageTest, and leave some non-working values here. Currently _BaseTest seems to be a mix of helper utilities, default values, and 1 test method. It may also require moving test_dont_cache somewhere else as well.

I've noticed it because of removal of FilesystemStorageTest - it was not immediately clear where FilesystemCacheStorage is tested  - you go to the direct parent (DefaultStorageTest), and there is not storage_class set.

I haven't really checked the code though :)

Reply:
Yeah, I created #6649 after looking at this and trying to fix it, this is just a small simple fix for a part of the problem. E.g. simply moving these values from `_BaseTest` to `DefaultStorageTest` will break `DummyPolicyTest` that subclasses it directly. And `test_dont_cache` seems to be useful to run for all 4 combinations of options.